docs/contributing_to_qiskit.rst

```diff
@@ -744,126 +138,91 @@ environments (e.g. one per supported Python version, for older versions of Qiski
-*****************
+Pull Requests
+=============
 
-The Qiskit project is made up of several elements each performing different
-functionality. Each is independently useful and can be used on their own,
-but for convenience we provide this repository and meta-package to provide
-a single entrypoint to install all the elements at once. This is to simplify
-the install process and provide a unified interface to end users. However,
-because each Qiskit element has its own releases and versions, some care is
-needed when dealing with versions between the different repositories. This
-document outlines the guidelines for dealing with versions and releases of
-both Qiskit elements and the meta-package.
+We use `GitHub pull requests
+<https://help.github.com/articles/about-pull-requests>`__ to accept
+contributions.
 
-For the rest of this guide the standard Semantic Versioning nomenclature will
-be used of: ``Major.Minor.Patch`` to refer to the different components of a
-version number. For example, if the version number was ``0.7.1``, then the major
-version is ``0``, the minor version ``7``, and the patch version ``1``.
+While not required, opening a new issue about the bug you're fixing or the
+feature you're working on before you open a pull request is an important step
+in starting a discussion with the community about your work. The issue gives us
+a place to talk about the idea and how we can work together to implement it in
+the code. It also lets the community know what you're working on, and if you
+need help, you can reference the issue when discussing it with other community
+and team members.
 
+If you've written some code but need help finishing it, want to get initial
+feedback on it prior to finishing it, or want to share it and discuss prior
+to finishing the implementation, you can open a *Draft* pull request and prepend
+the title with the **\[WIP\]** tag (for Work In Progress). This will indicate
+to reviewers that the code in the PR isn't in its final state and will change.
+It also means that we will not merge the commit until it is finished. You or a
+reviewer can remove the [WIP] tag when the code is ready to be fully reviewed for merging.
 
-Meta-package Version
-====================
+Before marking your Pull Request as "ready for review" make sure you have followed the
+PR Checklist below. PRs that adhere to this list are more likely to get reviewed and
+merged in a timely manner.
 
-The Qiskit meta-package version is an independent value that is determined by
-the releases of each of the elements being tracked. Each time we push a release
-to a tracked component (or add an element) the meta-package requirements, and
-version will need to be updated and a new release published. The timing should
-be coordinated with the release of elements to ensure that the meta-package
-releases track with element releases.
+.. _pr-checklist:
 
-Adding New Tracked Elements
+**Pull Request Checklist:**
 ---------------------------
```

Comment:
This turns the HTML into `<h4><strong>Pull Request Checklist:</strong></h4>`, whereas I guess the intent is more for us to fix our CSS here?

Reply:
I just wanted to make it bold because it wasn't very clear just as h4 (at least to my eyes)

Reply:
i don't have a strong opinion on how it should look or how it should be implemented, it just looked better to me imho

Reply:
whether we adjust the HTML or the CSS i don't think it matters much for this site

Reply:
Yeah, agreed